### PR TITLE
Fixes for Ploopy devices

### PR DESCRIPTION
**Pull request by ploopyco**
## Description

The DPI code for the Ploopy Nano and Mini was not functioning due to a bug with the ADNS serial write function, as well as with the ADNS initialization routine. This PR fixes both issues.

The backwards and forwards button on the Mini were reversed; this is fixed in this PR.

This PR also addresses the fact that some of the PCB revisions for the Mini and Classic are updated, so READMEs and directories have been changed to make this a bit more obvious.

## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by drashna** [comment]
Could you separate out the adns fix from the folder moves? 

In general, folder moves require that he PR target the `develop` branch and go through the breaking changes process.  which means it wouldn't hit master for a few months. 

Alternative, I can see about pulling out the bug fixes and add them to #13043

**Comment by ploopyco** [comment]
I'll pull the folder moves out.

**Comment by ploopyco** [comment]
Fixed; no breaking changes are included anymore.

**Review by drashna — APPROVED** [review]

File: keyboards/ploopyco/trackball_mini/trackball_mini.c [hunk 1]
```diff
@@ -225,7 +235,4 @@ void matrix_init_kb(void) {
 }
 
 void keyboard_post_init_kb(void) {
```
Comment (fauxpark) [inline keyboards/ploopyco/trackball_mini/trackball_mini.c:237]:
```suggestion
```

File: keyboards/ploopyco/trackball_mini/trackball_mini.c [hunk 2]
```diff
@@ -225,7 +235,4 @@ void matrix_init_kb(void) {
...
-    adns_set_cpi(dpi_array[keyboard_config.dpi_config]);
-
-    keyboard_post_init_user();
 }
```
Comment (fauxpark) [inline keyboards/ploopyco/trackball_mini/trackball_mini.c:238]:
```suggestion
```

File: keyboards/ploopyco/trackball_mini/trackball_mini.c [hunk 3]
```diff
@@ -18,6 +18,8 @@
  */
 
 #include "trackball_mini.h"
+#include "wait.h"
+#include "print.h"
```
Comment (fauxpark) [inline keyboards/ploopyco/trackball_mini/trackball_mini.c:22]:
```suggestion
#include "debug.h"
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by nooges — APPROVED** [review]

**Pull request merged**